google-cloud-storage/test/helper.rb

```diff
@@ -241,13 +241,15 @@ def patch_bucket_args if_metageneration_match: nil,
                         if_metageneration_not_match: nil,
                         predefined_acl: nil,
                         predefined_default_object_acl: nil,
-                        user_project: nil
+                        user_project: nil,
+                        options: {}
```

Comment:
I see you updated mocks for existing tests; is it possible to also add a simple test with preconditions to ensure that retries do occur in this case? I guess this will be covered by conformance tests as well.

Reply:
Yup conformance tests should cover it